docs/interfaces.md

```diff
@@ -460,6 +460,13 @@ processes on the system.
 
 * Auto-Connect: no
 
+### system-power-control
```

Comment:
Actually, this might well be "reboot' simply. Let me catch up with @jdstrand about that.